submissions/lisnyak-o/popup-kottans/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="HandheldFriendly" content="true">
+  <meta name="MobileOptimized" content="width">
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="shortcut icon" href="favicon.png" type="image/x-icon">
+  <link
+    href="https://fonts.googleapis.com/css?family=Fira+Sans:100,100italic,200,200italic,300,300italic,regular,italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic"
+    rel="stylesheet" />
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/reset.css">
+  <title>Popup HTML&CSS</title>
+</head>
+
+<body>
+  <header class="header">
+    <div class="container">
+      <nav class="menu">
+        <ul class="menu__list menu__wrapper">
+          <li class="menu__item"><a href="#" class="menu__item-link">Mail</a></li>
+          <li class="menu__item"><a href="#" class="menu__item-link">Picture</a></li>
+          <li class="menu__item">
+            <a href="#" class="menu__item-link">
+              <label class="menu__item-btn" for="popup-checkbox">
+                <img class="menu__item-img" src="/images/popup-button.png" alt="popup-btn">
+              </label>
+            </a>
+          </li>
+          <li class="menu__item">
+            <a href="#" class="menu__item-link">
+              <img class="menu__item-img" src="/images/bell.png" alt="bell-btn">
+            </a>
+          </li>
+          <li class="menu__item">
+            <a href="#" class="menu__item-link">
+              <img class="menu__item-photo" src="/images/photo.jpg" alt="photo">
+            </a>
+          </li>
+        </ul>
+      </nav>
+      <div class="popup">
+        <input type="checkbox" class="popup-btn" id="popup-checkbox">
+        <div class="popup-base content">
+          <div class="square"></div>
+          <label class="close-btn" for="popup-checkbox"></label>
+          <div class="popup-base__scroll">
+            <div class="scroll__items">
+              <a href="#" class="scroll__item">
```

Comment:
Not informative name of the class element, maybe `submenu-items` and then later `extramenu-items` is was the more correctly name

Reply:
Coming up with classes is always fun :)